core/domain/feature_flag_services_test.py

```diff
@@ -501,8 +509,11 @@ def test_feature_flag_enabled_for_users_in_5_perc_when_increased_to_10_perc(
                     count_feature_flag_enabled_for_10_perc += 1
                     user_ids_for_which_feature_flag_enabled_for_10_perc.add(
                         user_id)
+            # (For detail explanation please look at above comment)For p = 0.1,
+            # normal distribution with mean np = 100 and standard deviation
```

Comment:
For p = 0.1, the binomial random variable follows an approximate normal distribution with mean np = 100 and standard deviation 9.48. The range would be ...

(Btw, strictly speaking, it's probably 9.49.)

Reply:
Done